src/main/java/frc/robot/Robot.java

```diff
@@ -19,19 +22,32 @@
 public class Robot {
 
 	public static final RobotType ROBOT_TYPE = RobotType.determineRobotType();
+	private final MultiAprilTagVisionSources aprilTagVisionSources;
 
 	public Robot() {
 		BatteryUtils.scheduleLimiter();
+		// waiting for swerve and poseestimator to be merged in order to have botpose2 working
+		this.aprilTagVisionSources = new MultiAprilTagVisionSources(
+			VisionConstants.MULTI_VISION_SOURCES_LOGPATH,
+			() -> Rotation2d.fromDegrees(0), // swerve::getGyroAbsoluteYaw,
+			() -> Rotation2d.fromDegrees(0), // () -> poseEstimator.getEstimatedPose().getRotation(),
+			VisionConstants.DEFAULT_VISION_SOURCES
+		);
 	}
 
 	public void periodic() {
 		BatteryUtils.logStatus();
 		BusChain.logChainsStatuses();
 		CommandScheduler.getInstance().run();
+		aprilTagVisionSources.periodic();
```

Comment:
above command schedular

Reply:
done. Why isn't there a comment to indicate that like there is in RobotPeriodic (in robotManager)?